geps/gep-713/index.md

```diff
@@ -1,1157 +1,747 @@
 
-Ana’s phone shrills while she’s typing, and she drops it. When she picks it
-up again she sees a stack of alerts. She goes pale as she quickly flips
-through them: there’s one for every single service in the `baker` namespace.
+The process of calculating Effective meta resources (Effective policies) consists of walking the hierarchy of target objects, from most specific to least specific (i.e., bottom-up, from the leaves towards the roots of the DAG of target objects) or from least specific to most specific (top-down), map reducing to a single meta resource spec each pair of meta resources adjacent to each other in the hierarchy, applying at each step one of the supported merge strategies described below, until no more than one spec remains for each effective target.
 
-**Ana**: _PUT IT BACK!!_
+Meta resource kinds that implement more than one merge strategy MUST provide fields for the instances of the meta resource to specify a chosen strategy (described in the next section). The least specific meta resource of the pair of meta resources whose specs are merged into one dictates the merge strategy to apply in such cases.
 
-**Chihiro**: _Just did. Be glad you couldn't hear all the alarms here._ 😕
+#### Conflict resolution
 
-**Ana**: _What the hell just happened??_
+If multiple meta resources target the same context, this is considered to be a conflict.
 
-**Chihiro**: _At a guess, all the workloads in the `baker` namespace actually
-fail a lot, but they seem OK because there are retries across the whole
-namespace?_ 🤔
+Conflicts must be resolved by applying a defined *merge strategy* (see further definition in the next section), where the meta resource considered higher between two conflicting specs dictates the merge strategy according to which the conflict must be resolved, defaulting to the lower spec (more specific) beating the higher one if not specified otherwise.
 
-Ana's blood runs cold.
+The following criteria MUST be applied to determine which meta resources are higher or lower and thus infer which merge strategy to apply to calculate the effective spec, continuing on ties:
 
-**Chihiro**: _Yeah. Looking a little closer, I think your `baker` rollout this
-morning would have failed without those retries._ 😕
+1. Between two meta resources at different levels of the hierarchy, the one attached higher in the hierarchy MUST be considered higher than the one attached lower in the hierarchy.
+2. Between two meta resources at the same level of the hierarchy, the older meta resource based on creation timestamp MUST be considered higher than the newer one.
+3. Between two meta resources at the same level of the hierarchy and identical creation timestamps, the meta resource appearing first in alphabetical order by `{namespace}/{name}` MUST be considered higher than the other.
```

Comment:
```suggestion
1. Between two meta resources at different levels of the hierarchy, the one attached higher wins.
2. Between two meta resources at the same level of the hierarchy, the older meta resource based on creation timestamp wins.
3. Between two meta resources at the same level of the hierarchy and identical creation timestamps, the meta resource appearing first in alphabetical order by `{namespace}/{name}` wins.
```

Reply:
Done with minor addition, again to avoid confusion between choosing a merge strategy versus what spec ends up winning according to the merge strategy that was chosen.